### DISCUSSION TITLE
How can I set a default value on String[] field?

### DISCUSSION THREAD

**Discussion by lumenwrites** [prisma/prisma#10763] [Q&A] [answered]
In my prisma schema I need to set a field to be something like this:
```
  writingDays         String[] @default(["Mon", "Tue", "Wed", "Thu", "Fri"])
```

But VSCode tells me:
```
Error parsing attribute "@default": Cannot set a default value on list field.
```

What should I do? I do need some way to set the defaults on this field.

**Comment by janpio** [comment, marked as answer]
This is possibly the related feature request to this not being possible yet: https://github.com/prisma/prisma/issues/10773

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
